src/lib/Microsoft.Fx.Portability.Reports.Json/JsonReportWriter.cs

```diff
@@ -5,29 +5,25 @@
 using Microsoft.Fx.Portability.Reporting;
 using Newtonsoft.Json;
 using System.IO;
+using System.Threading.Tasks;
 
 namespace Microsoft.Fx.Portability.Reports
 {
     public class JsonReportWriter : IReportWriter
     {
-        private readonly ResultFormatInformation _formatInformation;
-
         public JsonReportWriter()
         {
-            _formatInformation = new ResultFormatInformation
+            Format = new ResultFormatInformation
             {
                 DisplayName = "Json",
                 MimeType = "application/json",
                 FileExtension = ".json"
             };
         }
 
-        public ResultFormatInformation Format
-        {
-            get { return _formatInformation; }
-        }
+        public ResultFormatInformation Format { get; }
 
-        public void WriteStream(Stream stream, AnalyzeResponse response)
+        public Task WriteStreamAsync(Stream stream, AnalyzeResponse response)
```

Comment:
Nice to have: useless async change.

Reply:
It was for IReportWriter.WriteStream to IReportWriter.WriteStreamAsync